InternetRadioDevice/InternetRadio/Interfaces/AllJoyn/AllJoynInterfaceManager.cs

```diff
@@ -1,15 +1,11 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
-
-using com.microsoft.maker.InternetRadio;
 using Windows.Devices.AllJoyn;
 using Windows.Foundation;
-using System.ComponentModel;
 using Windows.ApplicationModel;
 using System.Xml.Linq;
+using com.microsoft.maker.InternetRadio;
```

Comment:
Where is this namespace defined? We should change it to `Microsoft.Maker.InternetRadio` for consistency with the rest of the `Microsoft.Maker` namespaces